src/vs/workbench/browser/parts/editor/editorPicker.ts

```diff
@@ -267,4 +267,4 @@ export class AllEditorsPicker extends BaseEditorPicker {
 
 		return super.getAutoFocus(searchValue);
 	}
-}
```

Comment:
@bpasero sorry about the newlines.  I can rebase them out. should we add `"files.insertFinalNewline": false` to workspace settings or `.editorconfig` ?